.github/workflows/tester.yml

```diff
@@ -0,0 +1,64 @@
+name: Tester
+
+on:
+  push:
+    paths:
+      - '.github/**'
+      - 'app/**'
+      - 'lib/**'
+      - 'regression-tests/**'
+      - 'src/**'
+      - 'test/**'
+      - 'unit-tests/**'
+  pull_request:
+    paths:
+      - '.github/**'
+      - 'app/**'
+      - 'lib/**'
+      - 'regression-tests/**'
+      - 'src/**'
+      - 'test/**'
+      - 'unit-tests/**'
+
+jobs:
+  test:
+    name: Build and test sslang
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-18.04]
```

Comment:
Is there a reason this uses 18.04? Why not 20.04 (or whatever the latest LTS is)?

Reply:
No particular reason; I just chose it randomly xD. Should we change it to `ubuntu-latest` or `ubuntu-20.04` (explicitly fix the version)?